pimd/pim_cmd.c

```diff
@@ -8394,11 +8397,12 @@ DEFUN (interface_ip_igmp_last_member_query_interval,
 
 DEFUN (interface_no_ip_igmp_last_member_query_interval,
        interface_no_ip_igmp_last_member_query_interval_cmd,
-       "no ip igmp last-member-query-interval",
+       "no ip igmp last-member-query-interval [(1-65535)]",
        NO_STR
        IP_STR
        IFACE_IGMP_STR
-       IFACE_IGMP_LAST_MEMBER_QUERY_INTERVAL_STR)
+       IFACE_IGMP_LAST_MEMBER_QUERY_INTERVAL_STR
+       IGNORED_IN_NO_STR)
 {
 	char default_last_member_query_count[4];
```

Comment:
Same here: either stringnify define or use `DESTROY` in the YANG op.

Reply:
default_robustness